source/usage-examples/watch.txt

```diff
@@ -4,7 +4,54 @@ Watch for Changes
 
 .. default-domain:: mongodb
 
-You can keep track of changes to data in MongoDB, such as changes to a
-collection, database, or deployment, by opening a **change stream**. A change
-stream allows applications to watch for changes to data and react to them.
-The change stream returns **change event** documents when changes occur.
+You can open a change stream on a ``MongoCollection``,
+``MongoDatabase``, or ``MongoClient`` by using the ``Watch()`` method.
+
+The following example passes an aggregation pipeline to the ``Watch()``
+method, which opens a change stream on the ``haikus`` collection and
+returns the most current majority-committed version of the updated
+document for replace operations:
+
+.. include:: /includes/usage-examples/run-example-tip.rst
+
+.. literalinclude:: /includes/usage-examples/code-snippets/watch.go
+   :start-after: begin watch
+   :end-before: end watch
+   :emphasize-lines: 3
+   :language: go
+   :dedent:
+
+Click `here <{+example+}/watch.go>`__ to see a fully runnable example.
+
+Expected Result
+---------------
+
+After you run the preceding example, then run the :doc:`Replace a
+Document usage example </usage-examples/replaceOne>`, you should see
```

Comment:
```suggestion
After you start the preceding example, run the :doc:`Replace a
Document usage example </usage-examples/replaceOne>` in a different shell while the above code snippet is running. Once you have run the other usage example, you should see
```

Issue: I think this should be explicit that you need two shells to complete this example. I think it is hard to understand that you need two shells to produce output unless you are familiar with what a change stream is. My suggestion is a little messy but I think it communicates what I think should be made explicit:

- You need two shells
- Both shells need to be running simultaneously